imdb_django/management/commands/import_akas.py

```diff
@@ -0,0 +1,84 @@
+
+            with transaction.atomic():  # Use a transaction for database consistency
+                row_count = 0  # Initialize a row count
+
+                for row in tsv_reader:
+                    title_id = row["titleId"]
+                    ordering = row["ordering"]
+                    title = row["title"]
+                    region = row["region"]
+                    language = row["language"]
+                    types = row["types"]
+                    attributes = row["attributes"]
+                    is_original_title = row["isOriginalTitle"] == "1"
+
+                    try:
+                        title_aka, created = TitleAka.objects.get_or_create(
+                            titleId_id=title_id,
+                            ordering=ordering,
+                            title=title,
+                            region=region,
+                            language=language,
+                            types=types,
+                            attributes=attributes,
+                            isOriginalTitle=is_original_title,
+                        )
+                    except IntegrityError:
+                        self.stdout.write(
+                            self.style.SUCCESS(f"Loaded row {row_count}:")
+                        )
+                        self.stdout.write(f"titleId: {title_id}")
+                        self.stdout.write(f"ordering: {ordering}")
+                        self.stdout.write(f"title: {title}")
+                        self.stdout.write(f"region: {region}")
+                        self.stdout.write(f"language: {language}")
+                        self.stdout.write(f"types: {types}")
+                        self.stdout.write(f"attributes: {attributes}")
+                        self.stdout.write(f"isOriginalTitle: {is_original_title}\n")
+
+                        # Handle the case where the foreign key reference fails
+                        self.stderr.write(
+                            self.style.ERROR(
+                                f"Failed to load row {row_count + 1}: ForeignKey reference to Title failed"
+                            )
+                        )
+                        continue
+
+                    row_count += 1  # Increment row count
+
+                    # Print the data for the current row
+                    self.stdout.write(self.style.SUCCESS(f"Loaded row {row_count}:"))
+                    self.stdout.write(f"titleId: {title_id}")
+                    self.stdout.write(f"ordering: {ordering}")
+                    self.stdout.write(f"title: {title}")
+                    self.stdout.write(f"region: {region}")
+                    self.stdout.write(f"language: {language}")
+                    self.stdout.write(f"types: {types}")
+                    self.stdout.write(f"attributes: {attributes}")
+                    self.stdout.write(f"isOriginalTitle: {is_original_title}\n")
```

Comment:
Where are we printing this data? 

On terminal for logging purposes! If this is the case then we should use the [Django logger](https://docs.djangoproject.com/en/4.2/topics/logging/#topic-logging-parts-loggers)

_Please keep in mind, logging in such cases (like parsing files) considerably reduces the speed_

Reply:
yes sir, i am using it to see what data has been loaded into the files. Will change it to use django logger.